webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -506,6 +506,127 @@ <h3 class="page-title">Logging</h3>
+          <div class="col-xs-12 callout-fade callout-warning">
+            <div class="marker">
+              <span class="glyphicon glyphicon-info-sign"></span>
+            </div>
+            <h4>Trigger immidiate update of nodes when their configuration changes</h4>
+            <p>When new policies are generated, you have to wait for the next scheduled run of a node to have it apply them.</p>
+            <p>
+              Depending of your use case, you may prefer to reduce latency between a node configuration change and its application on
+              the node. For that, Rudder can send an update notification at the end of a policy generation to all nodes whose configuration
+              was changed. That notification will tell the corresponding nodes to start a run immediately - and thus, not honouring neither
+              their agent run period nor the splaytime between nodes. This can be a problem on large installation where hundreds of nodes would start a
+              run concurrently, using a lot of CPU resources and network at the same time.
+            </p>
+            <p>
+              So by default, we advice to always limit the notification to a maximum number of nodes. Setting that number
+              to 0 has the same effect than disabling that hook.
+            </p>
+            <p>
+              You can also limit the number of notified nodes to a fraction of the total updated node so that you can
+              see on a small number of nodes if a change works. This is especially intereting if your schedule period is
+              big, like 30 min, which let you some time to revert a faulty change. Of course, that help does not replace
+              a real rollout strategy to deploy policies, and you should always test critical configuration changes on a
+              dedicated group of nodes before rolling out the change on other production nodes.
+            </p>
+            <p>
+              If the number of nodes given by the ratio is bigger than the maximun number of nodes configured in the previous
+              parameter, then the configured maximun number of node is used.
+            </p>
+            <p>
+              Nodes are chosen at random, with the exception of the policy servers that will always be notified first.
+            </p>
+            <p>
+              Setting the ration is given in percent of node to update, rounded up, so that 1% of 10 nodes will be 1 node,
+              and 90% of 8 nodes will be all the 8 nodes. Setting that ration to 0 is the same than disabling that hook,
+              and setting it to 100
+            </p>
+            <p>
+              The notification use the "trigger remote agent run" feature and you will need to open network port accordingly to
+              that (<a href="https://docs.rudder.io/reference/current/installation/requirements.html#configure-the-network">see documentation</a>).
```

Comment:
shouldn't it be a link to the local doc ?

Reply:
right